interfaces/Portalicious/src/app/pages/project/project-registrations/components/custom-message-preview/custom-message-preview.component.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  computed,
+  input,
+} from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+import { InputTextareaModule } from 'primeng/inputtextarea';
+
+import { SendMessageData } from '~/domains/registration/registration.api.service';
+
+@Component({
+  selector: 'app-custom-message-preview',
+  standalone: true,
+  imports: [InputTextareaModule, FormsModule],
+  templateUrl: './custom-message-preview.component.html',
+  styles: ``,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class CustomMessagePreviewComponent {
+  sendMessageData = input<SendMessageData>();
+  // TODO: AB#30847 add additional property for dummy data to be used in the preview
+
+  messageText = computed(() => {
+    const data = this.sendMessageData();
+
+    if (data === undefined) {
+      return '';
+    }
+
+    // TODO: AB#30847 replace placeholders in message with dummy data
+    if ('customMessage' in data) {
+      return data.customMessage;
+    }
+
+    return data.messageTemplateKey;
```

Comment:
You're probably taking care of this in AB#30847 to actually replace the key with the english contents of the template. If so, this comment can be resolved :)

Reply:
I don't know why it marked it like this, I only clicked line 37. Im specifically talking about line 37 here.

Reply:
Correct, already taken care of in #5968 as a matter of fact 😄